homeassistant/components/binary_sensor/zha.py

```diff
@@ -26,6 +26,7 @@
     0x002b: 'gas',
     0x002d: 'vibration',
 }
+DEVICE_CLASS_OCCUPANCY = 'occupancy'
```

Comment:
Any guidelines if I open another PR to add them?

Reply:
The sensor component imports them from const.py, but I think it would be better to define them in the binary_sensor component. Technically we should make an architecture issue. Maybe just do that and see if anyone opposes it. Then we can go ahead.